lib/gems/pending/appliance_console/cli.rb

```diff
@@ -182,13 +182,18 @@ def run
 
     def set_db
       raise "No encryption key (v2_key) present" unless key_configuration.key_exist?
+      raise "Must provide a password to set database" unless password?
```

Comment:
I think we can improve this sentence a bit ...

Maybe "A password is required to configure a database"

Reply:
👍